src/plane.ts

```diff
@@ -0,0 +1,38 @@
+// todo:
+// - pitch ctrl derivs
+
+import * as mathjs from 'mathjs';
+import * as THREE from 'three';
+
+export class Plane {
+  wingspan: number;
+  length: number;
+  chord: number;
```

Comment:
If wingspan, length, and chord will only ever be used to set up the model you may not need to store these in the class. An argument for keeping them is that they could be useful if you need to figure out which Plane you're looking at when attaching a debugger or doing console logging.

Production code may be dragged down by an extra copy of the value being kept around, but production code would probably have some optimizations applied to remove unreferenced variables.

Not suggesting a particular action here, just expounding

Reply:
That's very fair, and these properties may become vestigial when I implement better geometry generation, anyways. I'll leave them for now, some corner of my brain is saying they'll be useful for physics simultaion.